packages/client/hmi-client/src/components/widgets/tera-related-documents.vue

```diff
@@ -153,8 +161,52 @@ const sendToAlignModel = async () => {
 
 		aligning.value = false;
 		emit('enriched');
+		getRelatedDocuments();
 	}
 };
+
+const openAsset = (assetRoute: AssetRoute) => {
+	router.push({
+		name: RouteName.Project,
+		params: assetRoute
+	});
+};
+
+onMounted(() => {
+	getRelatedDocuments();
+});
+
+watch(
+	() => props.assetId,
+	() => {
+		getRelatedDocuments();
+	}
+);
+
+async function getRelatedDocuments() {
+	if (!props.assetType) return;
+	let provenanceType;
+	if (props.assetType === ResourceType.MODEL) {
+		provenanceType = ProvenanceType.Model;
+	}
+	if (props.assetType === ResourceType.DATASET) {
+		provenanceType = ProvenanceType.Dataset;
+	}
```

Comment:
I think we still want to pass the ProvenanceType here since we could have much more than just ResourceTypes being passed into the `getRelatedArtifacts` function.  I have instead made this better by making a sepatate `mapResourceTypeToProvenanceType` function to make this more readable